.github/workflows/docker-backend.yml

```diff
@@ -9,19 +9,13 @@ on:
       # Name of service. Empty for root service, needs to be unique for services from subfolders. 
       # Will be used as a last part of container path and matched with version tags
       name:
-        required: false
+        required: true
         type: string
-        default: ""
 
       path:
         required: false
         type: string
         default: "."
-        
-      repository:
-        required: false
-        type: string
-        default: ${{ github.repository }}
```

Comment:
Wait, why did you remove this, when it is used in the `IMAGE_NAME` still?

Reply:
Oh? I was searching that variable ;_;

Reply:
Wait, it doesn't, IMAGE_NAME only uses registry ^_^

Reply:
oh oops